src/pages/about.js

```diff
@@ -1,34 +0,0 @@
-import React from 'react';
```

Comment:
This file should have been renamed to `brand.js`not deleted

Reply:
Yeah I renamed "about.js" to "brand.js" and then did all the changes.
Did I go wrong somewhere?